src/tab/Tab.tsx

```diff
@@ -16,20 +16,26 @@ export interface TabProps {
   items: TabItem[];
   children: React.ReactNode[];
   testid?: string;
-  initialActiveTabIndex?: number;
+  selectedItem?: number;
```

Comment:
```suggestion
  activeTabIndex?: number;
```
I think this would be more descriptive, because we are passing the index rather than the item itself.

This causes duplicate names, you can solve it by renaming this while destructring in line 27:
```
activeTabIndex: activeTabIndexFromProps = 0
```